src/config/env.js

```diff
@@ -1,22 +1,27 @@
 require("dotenv-flow").config();
 
-module.exports = Object.freeze({
+const config = Object.freeze({
     // Database
     db_host: process.env.DB_HOST,
     db_port: process.env.DB_PORT,
-    db_uri: process.env.DB_URI,
+    db_name: process.env.DB_NAME,
+    db_uri: process.env.DB_URI ||
+    (
+        process.env.DB_HOST
+            && process.env.DB_PORT
+            && process.env.DB_NAME
+            && `mongodb://${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_NAME}`
+    ),
```

Comment:
If we are to go with this solution we should just make a method for this, I think it would be simpler or at least abstract from the rest of the configuration.